posthog-core/src/index.ts

```diff
@@ -1773,7 +1779,7 @@ export abstract class PostHogCore extends PostHogCoreStateless {
     ) as PostHogFeatureFlagDetails
   }
 
-  private getKnownFeatureFlags(): PostHogDecideResponse['featureFlags'] | undefined {
+  protected getKnownFeatureFlags(): PostHogDecideResponse['featureFlags'] | undefined {
```

Comment:
easier to reuse this method that does all normalization otherwise I'd need to duplicate some code on the react native side or expose way more internal methods to the react native package